.github/workflows/main.yml

```diff
@@ -243,3 +243,19 @@ jobs:
 
       - name: run check
         run: make itest
+
+  ########################
+  # check PR updates release notes
+  ########################
+  release-notes-check:
+    name: check release notes updated
+    runs-on: ubuntu-latest
+    if: '!contains(github.event.pull_request.labels.*.name, ''no-changelog'')'
+    steps:
+      - name: git checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
```

Comment:
Why do we need the `fetch-depth: 0` here, but not in lnd?

Reply:
i just used what is used else-where in lit

Reply:
removed